### ISSUE TITLE
Sandbox not functioning as expected

### ISSUE THREAD

**Issue by defionscode** [architect/architect#581]
**Describe the issue**
Following the Quick Start guidance proves unfruitful for sandbox mode on macOS

**Steps to reproduce**
**environment**
```
node --version
v10.16.3

which node
/Users/jdavila/.nvm/versions/node/v10.16.3/bin/node

which arc
/Users/jdavila/.nvm/versions/node/v10.16.3/bin/arc
```

**.arc**
```
@app
init

@http
get /

@aws
runtime nodejs10.x
profile default
region us-east-1
bucket arc-codes-testing
```

**Steps**
1. `arc sandbox` which produces
    ```
    $ arc sandbox
       App ⌁ init
       Region ⌁ us-east-1
       Profile ⌁ default
       Version ⌁ Architect 6.0.28
       cwd ⌁ /Users/jdavila/Repos/arc-tinker

      ✓ Sandbox Found Architect manifest, starting up
      ✓ Sandbox Project files hydrated into functions
      ✓ Sandbox Loaded routes
          get / ................................. get-index

      ✓ Sandbox Sandbox started in 60ms
      ✈︎ Local environment ready!

          http://localhost:3333
    ```
1. Then navigate to `http://localhost:3333` in the browser
1. Get generic Not Found browser page
1. Terminal displays
    ```
    /Users/jdavila/.nvm/versions/node/v10.16.3/lib/node_modules/@architect/architect/node_modules/@architect/sandbox/src/invoke-lambda/index.js:48
         runtimes[runtime](options, timeout, callback)
                       ^

    TypeError: runtimes[runtime] is not a function
        at done (/Users/jdavila/.nvm/versions/node/v10.16.3/lib/node_modules/@architect/architect/node_modules/@architect/sandbox/src/invoke-lambda/index.js:48:24)
        at done (/Users/jdavila/.nvm/versions/node/v10.16.3/lib/node_modules/@architect/architect/node_modules/@architect/sandbox/src/invoke-lambda/get-config.js:18:7)
        at FSReqWrap.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:53:3)
    ```

**Expected behavior**
I would expect it to work as described in the docs

**Screenshots**
If applicable, add screenshots to help explain your problem

**Desktop**
Please complete the following information (if appropriate):
 - OS: MacOS Catalina 10.15.2
 - Browser: Firefox 
 - Version: 71.0

**Comment by defionscode** [comment]
I found the issue. It seems the default setting in `.arc` and `.arc-config` is

```
@aws
runtime node
# memory 1152
# timeout 30
# concurrency 1
```

when I change it to

```
@aws
runtime nodejs10.x
# memory 1152
# timeout 30
# concurrency 1
```

then it works.

Perhaps it makes sense to set the default to a value that _might_ work?

**Comment by ryanblock** [comment]
I know there was some work recently done in the runtime configuration code paths, cc @brianleroux

**Comment by brianleroux** [comment]
ya this is def my bad; thx for reporting! think the fix is in architect/create (somewhere)

**Comment by brianleroux** [comment]
fixed in `@architect/architect@6.1.0` !

**Comment by ryanblock** [comment]
Sorry @defionscode! Was actually fixed on the 7th, forgot to update this issue.
